outreach/sops/README.md

```diff
@@ -0,0 +1,41 @@
+# Operation Code Outreach Team
+
+Thanks for joining the Operation Code Outreach Team!
+
```

Comment:
Do you think it makes sense to list the current members of the Outreach team here also, or should that live somewhere else?

Reply:
That should probably live in some kind of alpha roster internal to the team. Probably best to keep SOPs separate.

Reply:
Gotcha. Would it be helpful for us to create some form of point of contact, like an outreach@operationcode.org email address, or a Slack group alias?

Reply:
A Slack alias makes sense. I could go either way on the team's email - I could see the need for one as external community involvement grows. But I can't imagine that email would have a lot of traffic in the very short term.